barserver/test/foo.test.ts

```diff
@@ -13,3 +18,13 @@ it('should get hello', async () => {
 it('should test that true === true', () => {
       expect(true).toBe(true)
 });
+
+it('should return',async ()=>{
+    const res= await request
+    .post('/api/postsecret')
+    .send({ data: JSON.stringify("secrettest") })
```

Comment:
"secrettest" 本身就是一个string，为啥还要JSON.stringify？
```suggestion
    .send({ data: "secrettest" })
```